app/src/main/java/org/gdg/frisbee/android/widget/FeedbackFragment.java

```diff
@@ -103,6 +105,14 @@ public void onCreate(Bundle savedInstanceState) {
         addProperty("loggedIn", PrefUtils.isSignedIn(getActivity())); // Optionally add some properties
         addProperty("appStarts", PrefUtils.getAppStarts(getActivity()));
         buildProperties();
+
+        setRetainInstance(true);
```

Comment:
I kinda had to. The problem is with the implementation of Doorbell's SDK we are using. 
They are dismissing the Dialog once the Send button is clicked and then they make the API request. In their Rest API class, they have a reference to Activity. 

So the problem is that achievement handler's method is run after `detachActivity` because of this. 

I wanted to change it completely. I wanted to make the dialog dismiss after API request finishes. Their callback only have `success` in it. We won't be notified if there is a network issue and the user will be stuck. 

I unfortunately couldn't figure out a better way because of this.